tests/test_core_urlproc.py

```diff
@@ -61,7 +59,11 @@ def test_get_user_agent():
     assert isinstance(user_agent, str)
 
 
-def test_check_response_status_code():
+@pytest.mark.parametrize(
+    "print_level", ["all", "files-with-urls-only",
```

Comment:
How is files-with-urls-only honored? I don't see it in the printing function, only the other four.

Reply:
so `files-with-urls-only` is not explicitly specified in the code, because I did not need to. Covering the other cases was sufficient. So -as you already noticed- you barely see the variable in the code, but if used, it works as expected.

Reply:
This doesn’t make any sense. Please provide an explicit example. If an option isn’t used I don’t understand why it exists.